src/popup/pages/FungibleTokens/TokenContainer.vue

```diff
@@ -166,33 +168,35 @@ export default defineComponent({
     const loading = ref<boolean>(true);
     const tokenPairs = ref({ token0: null, token1: null });
     const tokenBalances = useGetter<any[]>('fungibleTokens/tokenBalances');
-    const availableTokens = computed(() => root.$store.state.fungibleTokens.availableTokens);
+    const availableTokens = computed(
+      (): ITokenList => root.$store.state.fungibleTokens.availableTokens,
+    );
```

Comment:
If you'd like to you could use `useState` hook

Reply:
Done